swifttool/client.py

```diff
@@ -96,26 +97,51 @@ def __init__(self, data=None):
         self.replicas = 3
         self.min_part_hours = 1
         self.zones = {}
+        # Area to build new or update existing rings
```

Comment:
Please use __init__ for initialization only.  We should not be moving files around the __init__ method.